tests/test_logic.py

```diff
@@ -25,3 +25,12 @@ def test_prog_as_text(app: SphinxTestApp) -> None:
     app.build()
     outcome = (Path(app.outdir) / "index.txt").read_text()
     assert outcome == "magic - CLI interface\n*********************\n\n   magic\n"
+
+
+@pytest.mark.sphinx("text", testroot="noincludetitle")
+def test_noincludetitle_as_text(app: SphinxTestApp) -> None:
```

Comment:
I don't think you need here to define a new noincludetitle, you can just reuse basic 👍🏻